ci/docker/Dockerfile.centos8

```diff
@@ -0,0 +1,26 @@
+FROM centos:8 as base
+
+# CentOS 6 packages are no longer hosted on the main repository, instead they are in the CentOS Vault
+# RUN sed -i 's/^mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-Base.repo && \
+#     sed -i 's/#baseurl=http:\/\/mirror.centos.org\/centos\/$releasever/baseurl=http:\/\/vault.centos.org\/6.10/g' /etc/yum.repos.d/CentOS-Base.repo
```

Comment:
I think we can remove these from the file completely, since they only apply to CentOS 6